server/controllers/user.js

```diff
@@ -153,7 +153,16 @@ module.exports = {
       } else {
         const userId = req.user.id;
         const reviews = await Review.find({ userId: userId });
-        return res.status(201).json({ reviews });
+        const requests = await Request.find({
+          selectedReviewer: userId,
+        });
+        const embeddedReviewArray = [];
+
+        for (let i = 0; i < requests.length; i++) {
+          embeddedReviewArray.push(requests[i].embeddedReview);
+        }
```

Comment:
you might not need to create a variable for an array as **find** should return an array. you can use **select** to only extract the embeddedReview

Reply:
Interestingly, I tried .select("embeddedReview"), but ran into issues where the request was returned null.

I will retry that once more before merging.

Reply:
Ok, I tried using .select(), it doesn't return what I am trying to work with.